release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1676,6 +1719,8 @@ Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redha
 
 * Google Cloud Platform (GCP) UPI installation fails when the instance group name is longer than the maximum size of 64 characters. You are restricted in the naming process after adding the "-instance-group" suffix. Shorten the suffix to "-ig" to reduce the number of characters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921627[*BZ#1921627*])
 
+* For clusters that run on {rh-openstack} and use Kuryr, a bug in ovn-octavia-provider can cause load balancer listeners to be stuck in a `PENDING_UPDATE` state while the load balancer that they are attached to remains in an `ACTIVE` state. As a result, kuryr-controller can crash. To resolve this problem, update {rh-openstack} to a version that includes a fix as soon as possible. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019980[*BZ#20004542*])
```

Comment:
```suggestion
* For clusters that run on {rh-openstack} and use Kuryr, a bug in `ovn-octavia-provider` <noun_goes_here> can cause load balancer listeners to be stuck in a `PENDING_UPDATE` state while the load balancer that they are attached to remains in an `ACTIVE` state. As a result, the `kuryr-controller` <noun_goes_here> can crash. To resolve this problem, update {rh-openstack} to a version that includes a fix as soon as possible. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019980[*BZ#20004542*])
```

Reply:
```suggestion
* For clusters that run on {rh-openstack} and use Kuryr, a bug in the `ovn-octavia-provider` package can cause load balancer listeners to be stuck in a `PENDING_UPDATE` state while the load balancer that they are attached to remains in an `ACTIVE` state. As a result, `kuryr-controller` service instances can crash. To resolve this problem, update {rh-openstack} to a version that includes a fix as soon as possible. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2019980[*BZ#20004542*])
```

Reply:
@MaysaMacedo What do you think of this latest proposed change?

Reply:
LGTM

Reply:
Thanks!